File: wronggou-manager-web/src/main/java/com/wronggo/controller/SellerController.java
package com.wronggo.controller;

import com.wronggo.model.PageResult;
import com.wronggo.model.Result;
import com.wronggo.model.TbBrand;
import com.wronggo.model.TbSeller;
import com.wrongo.service.TbSellerService;
import org.apache.dubbo.config.annotation.Reference;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/seller")
public class SellerController {
    @Reference
    TbSellerService tbSellerService;
    @RequestMapping("/updateStatus")
    public Result updateStatus(String sellerId, String status){
        try {
            tbSellerService.updateStatus(sellerId, status);
            return new Result(true, "成功");
        } catch (Exception e) {
            e.printStackTrace();
            return new Result(false, "失败");
        }
    }
    @RequestMapping("/findPage")
    public PageResult findPage(int page, int rows){
        return tbSellerService.findPage(page, rows);
    }

    @RequestMapping("/findOne")
    public TbSeller findOne(String id){
        return tbSellerService.selectbyname(id);
    }
}

File: wronggou-manager-web/src/main/resources/static/js/controller/sellerController.js
app.controller('sellerController' ,function($scope,$controller,sellerService){

    $controller('baseController',{$scope:$scope});//继承
    //分页
    $scope.findPage=function(page,rows){
        sellerService.findPage(page,rows).success(
            function(response){
                $scope.list=response.rows;
                $scope.paginationConf.totalItems=response.total;//更新总记录数

            }
        );
    }

    //根据id查
    $scope.findOne=function(sellerId){
        sellerService.findOne(sellerId).success(
            function(response){
                $scope.entity= response;
            }
        );
    }

    $scope.updateStatus=function(sellerId,status){
        sellerService.updateStatus(sellerId,status).success(
            function(response){
                if(response.success){
                    $scope.reloadList();//刷新列表
                }else{
                    alert("失败");
                }
            }
        );
    }
});